core/templates/dev/head/services/searchService.js

```diff
@@ -67,16 +67,24 @@ oppia.factory('searchService', [
   var updateSearchFields = function(itemsType, urlComponent,
                                     selectionDetails) {
     var itemCodeGroup = urlComponent.match(/=\("[A-Za-z%20" ]+"\)/);
+    var itemCodes = itemCodeGroup ? itemCodeGroup[0] : null;
 
-    if (itemCodeGroup == undefined) {
+    var EXPECTED_PREFIX = '=("';
+    var EXPECTED_SUFFIX = '")';
+
+    if (!itemCodes ||
+        itemCodes.indexOf(EXPECTED_PREFIX) !== 0 ||
```

Comment:
IE... in that case, please ignore my subsequent comments

> On Apr 28, 2016, at 10:25 PM, Sean Lip <notifications@github.com> wrote:
> 
> In core/templates/dev/head/services/searchService.js:
> 
> >  
> > -    if (itemCodeGroup == undefined) {
> > +    var EXPECTED_PREFIX = '=("';
> > +    var EXPECTED_SUFFIX = '")';
> > +
> > +    if (!itemCodes ||
> > +        itemCodes.indexOf(EXPECTED_PREFIX) !== 0 ||
> IE has no support for startsWith, unfortunately ... https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>